firestore.rules

```diff
@@ -41,5 +41,9 @@ service cloud.firestore {
       allow read: if request.auth.uid != null && resource.data.userId == request.auth.uid;
       allow write: if false;
     }
+    match /customers/{userId} {
```

Comment:
Should this be called `cards`?

Reply:
we're storing the customer_ids so not sure it should